plugins/woocommerce/src/Internal/ComingSoon/ComingSoonRequestHandler.php

```diff
@@ -78,7 +78,11 @@ function () {
 		);
 
 		if ( ! empty( $coming_soon_template ) && file_exists( $coming_soon_template ) ) {
-			include $coming_soon_template;
+			if ( ! $is_fse_theme && $is_store_coming_soon ) {
+				echo get_the_block_template_html();
```

Comment:
This function is marked as private on https://developer.wordpress.org/reference/functions/get_the_block_template_html/, but it is the only function that I could find to render the template content without custom code.

I believe it is relatively safe to use, as it was introduced in WordPress 5.8.0."

Reply:
Nice find! I think this is a good solution. 🙌  How about we add a check to see if the function exists before calling it? This way, we can avoid any potential issues if the function is removed in the future.